.env

```
@@ -1,4 +1,7 @@
 DATABASE_URL="postgresql://neondb_owner:npg_Im7fQZ8sNUdX@ep-fancy-king-abfajg7o-pooler.eu-west-2.aws.neon.tech/neondb?sslmode=require&channel_binding=require"
 SESSION_SECRET=your-secret-key-here
 NODE_ENV=development
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The NODE_ENV key should go before the SESSION_SECRET key

(UnorderedKey)